Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterService.cs

```diff
@@ -202,11 +202,24 @@ public TwitterDataProvider Provider
         /// <returns>Strongly typed list of data returned from the service.</returns>
```

Comment:
This file has many warnings it would be great if you can fix it, like 
private TwitterDataProvider twitterDataProvider;